test/setup.ts

```diff
@@ -154,37 +154,37 @@ before("setup", async function () {
     }
 
     this.redeployBase = async function (permissioned: boolean) {
-        this.erc721 = <Extended<ERC721>>await deployExtendableContract(this.signers.admin, this.artifacts.erc721, [TOKEN_NAME, TOKEN_SYMBOL, this.extend.address, this.approve.address, this.baseGetter.address, this.onReceive.address, this.transfer.address, this.beforeTransfer.address]) 
+        this.erc721 = <Extended<ERC721>>await deployExtendableContract(this.signers.admin, this.artifacts.erc721, [TOKEN_NAME, TOKEN_SYMBOL, this.extend.address, this.approve.address, this.baseGetter.address, this.onReceive.address, this.transfer.address, this.hooks.address]) 
         this.tokenAsExtend = <ExtendLogic>await this.erc721.as(this.artifacts.extend);
         if (permissioned) await this.tokenAsExtend.extend(this.permissionedErc721MockExtension.address);
         else await this.tokenAsExtend.extend(this.erc721MockExtension.address);
     
         this.tokenAsApprove = <ApproveLogic>await this.erc721.as(this.artifacts.approve);
         this.tokenAsBurn = <BasicBurnLogic>await this.erc721.as(this.artifacts.burn);
         this.tokenAsBaseGetter = <GetterLogic>await this.erc721.as(this.artifacts.baseGetter);
-        this.tokenAsBeforeTransfer = <BeforeTransferLogic>await this.erc721.as(this.artifacts.beforeTransfer);
+        this.tokenAsHooks = <ERC721HooksLogic>await this.erc721.as(this.artifacts.hooks);
```

Comment:
I'm confused, where are these `tokenAsHooks` variables used?

Reply:
Ah, they aren't, so these variables might not be necessary since they're obviously hooks that are called internally between extensions.